cmd/soroban-rpc/internal/daemon/daemon.go

```diff
@@ -19,6 +24,53 @@ func Start(cfg config.LocalConfig) (exitCode int) {
 	logger := supportlog.New()
 	logger.SetLevel(cfg.LogLevel)
 
+	httpPortUint := uint(cfg.CaptiveCoreHTTPPort)
+	var captiveCoreTomlParams ledgerbackend.CaptiveCoreTomlParams
+	captiveCoreTomlParams.HTTPPort = &httpPortUint
+	captiveCoreTomlParams.HistoryArchiveURLs = cfg.HistoryArchiveURLs
+	captiveCoreTomlParams.NetworkPassphrase = cfg.NetworkPassphrase
+	captiveCoreTomlParams.Strict = true
+	captiveCoreToml, err := ledgerbackend.NewCaptiveCoreTomlFromFile(cfg.CaptiveCoreConfigPath, captiveCoreTomlParams)
+	if err != nil {
+		logger.WithError(err).Fatal("Invalid captive core toml")
```

Comment:
I think that you might want to exit here; the sooban-rpc wouldn't be operative without a captive core.

Reply:
It's a fatal log, it will exit